src/jira/extract-installation-from-jira-callback.ts

```diff
@@ -25,7 +26,7 @@ export const extractInstallationFromJiraCallback = async (req: Request, res: Res
 	const { jiraHost, clientKey } = installation;
 
 	req.addLogFields({
-		jiraHost,
+		jiraHost: createHashWithSharedSecret(jiraHost),
 		jiraClientKey: `${clientKey.substr(0, 5)}***}`
```

Comment:
We are already doing some obfuscation here with the JiraClientKey so left it as is.